<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoordinateSidebar</class>
 <widget class="QFrame" name="CoordinateSidebar">
  <property name="minimumWidth">
   <number>220</number>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <layout class="QVBoxLayout" name="sidebar_layout">
   <property name="spacing">
    <number>12</number>
   </property>
   <item>
    <widget class="QGroupBox" name="cursor_group">
     <property name="title">
      <string>Cursor position</string>
     </property>
     <property name="visible">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="cursor_form">
      <item row="0" column="0">
       <widget class="QLabel" name="cursor_x_label">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
     <item row="0" column="1">
       <widget class="QLineEdit" name="cursor_x">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>–</string>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
       </widget>
     </item>
      <item row="1" column="0">
       <widget class="QLabel" name="cursor_y_label">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
     <item row="1" column="1">
       <widget class="QLineEdit" name="cursor_y">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>–</string>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
       </widget>
     </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="flag_title">
     <property name="text">
      <string>Selected flag</string>
     </property>
     <property name="styleSheet">
      <string>font-weight: bold</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="flag_form">
     <item row="0" column="0">
      <widget class="QLabel" name="flag_x_label">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="flag_x">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>–</string>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="flag_y_label">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="flag_y">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>–</string>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="flag_hint">
     <property name="text">
      <string>Left click to drop/select flags.&#10;Right click a flag to remove it.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string>color: #bbbbbb; font-size: 11px</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="camera_title">
     <property name="text">
      <string>Track cameras</string>
     </property>
     <property name="styleSheet">
      <string>font-weight: bold</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="camera_list">
     <property name="minimumHeight">
      <number>120</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="TvModesPanel" name="tv_panel"/>
   </item>
   <item>
    <widget class="QLabel" name="camera_details_title">
     <property name="text">
      <string>Camera details</string>
     </property>
     <property name="styleSheet">
      <string>font-weight: bold</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="camera_details">
     <property name="text">
      <string>Select a camera to inspect.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignTop</set>
     </property>
     <property name="styleSheet">
      <string>font-size: 12px</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="coords_title">
     <property name="text">
      <string>World coordinates</string>
     </property>
     <property name="styleSheet">
      <string>font-weight: bold</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="CameraCoordinateTable" name="camera_table"/>
   </item>
   <item>
    <spacer name="sidebar_spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TvModesPanel</class>
   <extends>QWidget</extends>
   <header>track_viewer.tv_modes_panel</header>
  </customwidget>
  <customwidget>
   <class>CameraCoordinateTable</class>
   <extends>QTableWidget</extends>
   <header>track_viewer.camera_table</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
